packages/docs/components/modal.md

```diff
@@ -24,34 +22,57 @@ links:
 
 ## Anatomy
 
-<Description class="is-fpo">
+<Anatomy img="/images/anatomy-modal.svg" />
 
-Descriptive content around **modal anatomy** should go here.
+<Description>
 
-</Description>
+### Curtain
 
-<Anatomy img="/images/anatomy-modal.svg" />
+Covers the main content to provide focus to the information served in the dialog.
```

Comment:
Instead of "Covers", I think we should say "A dark overlay on top of" the main content…